### PR TITLE
docs: describe user invite process & cleanup

### PR DESCRIPTION
**Pull request by tomekpapiernik**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by vinckr — COMMENTED** [review]
The invite guide looks great , havent been able to try it out yet, but will next week.

For the other pages:
IMO we can remove the Tab entirely if there is only one thing in it.

**Comment by tomekpapiernik** [comment]
> For the other pages: IMO we can remove the Tab entirely if there is only one thing in it.

IDK, I kinda like the look even when there's a single tab. Can be handled in another PR, let's not block merging the invites doc.

File: docs/kratos/manage-identities/40_invite-users.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+4. The user performs account recovery which forces them to define new credentials.
+5. The user gets a fully functional account with the credentials they defined.
+
+:::info
+
+Automating this process requires writing custom code that relies on the specifics of your implementation.
+
+:::
+
+## Required API calls
```
Comment (fbolton) [inline docs/kratos/manage-identities/40_invite-users.mdx:26]: I was a bit confused by this subheading at first. Maybe consider changing it to something like "Procedure" or "Procedure to invite users"?

**Review by fbolton — CHANGES_REQUESTED** [review]
I have made one suggestion - otherwise, content looks good.

File: docs/kratos/manage-identities/40_invite-users.mdx (hunk 1 above)
Comment (tomekpapiernik) [reply docs/kratos/manage-identities/40_invite-users.mdx]: Changed to "Procedure" 👍🏻

**Review by fbolton — APPROVED** [review]

**Pull request merged**